### PR TITLE
chore: add generate challenge endpoints

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

File: seeds/endpointType.js [hunk 1]
```diff
@@ -192,15 +192,23 @@ const endpointTypes = [
...
   },
   {
     name: 'TP_CB_URL_CONSENT_PUT',
-    description: 'Participant callback URL where PUT /consent/{ID} can be sent'
+    description: 'Participant callback URL where PUT /consents/{ID} can be sent'
   },
   {
     name: 'TP_CB_URL_CONSENT_PUT_ERROR',
-    description: 'Participant callback URL where PUT consent/{ID}/error can be sent error information can be sent'
+    description: 'Participant callback URL where PUT /consents/{ID}/error can be sent error information can be sent'
```
Comment (lewisdaly) [inline seeds/endpointType.js:203]: nit: remove duplicate "can be sent"

File: seeds/endpointType.js [hunk 2]
```diff
@@ -192,15 +192,23 @@ const endpointTypes = [
...
+  },
+  {
+    name: 'TP_CB_URL_CONSENT_GENERATE_CHALLENGE_POST',
+    description: 'Participant callback URL where POST /consents/{ID}/generateChallenge can be sent'
+  },
+  {
+    name: 'TP_CB_URL_CONSENT_GENERATE_CHALLENGE_PUT_ERROR',
+    description: 'Participant callback URL where PUT /consents/{ID}/generateChallenge/error can be sent error information can be sent'
```
Comment (lewisdaly) [inline seeds/endpointType.js:211]: nit: remove duplicate "can be sent"

**Review by lewisdaly — CHANGES_REQUESTED** [review]

**Comment by kleyow** [comment]
@lewisdaly addressed nits.

**Review by eoln — APPROVED** [review]
lgtm

**Pull request merged**